server/test/write.go

```diff
@@ -1239,36 +1271,13 @@ var writeCommandTests = []writeCommandTest{
 		},
 		request: &openfgapb.WriteRequest{
 			Writes: &openfgapb.TupleKeys{TupleKeys: []*openfgapb.TupleKey{
-				tuple.NewTupleKey("document:budget", "reader", "*"),
+				tuple.NewTupleKey("document:budget", "reader", "group:*"),
 			}},
 		},
-		err: serverErrors.InvalidTuple("User '*' is not allowed to have relation reader with document:budget",
-			tuple.NewTupleKey("document:budget", "reader", "*"),
+		err: serverErrors.InvalidTuple("User 'group:*' is not allowed to have relation reader with document:budget",
+			tuple.NewTupleKey("document:budget", "reader", "group:*"),
 		),
 	},
-	{
```

Comment:
Removing this in favor of the added test in `server/test/write_authzmodel.go` that makes sure you can't write a 1.1 model without type information.